docs/hydro/eastern.rst

```diff
@@ -0,0 +1,31 @@
+Eastern
++++++++
+Profile generation for the Eastern Interconnection is slightly more complex than for
+the Western Interconnection. First, we handle pumped storage hydro (HPS) and
+conventional hydro (HYC) separately. Then, in addition to using the net demand profile
+to derive the profile of conventional hydro plants in some state as we did for
+California, historical hourly total hydro generation time series are also used in some
+regions.
+
+
++ **Pumped Storage Hydro**: We identified hydro plants in our grid model that
+  are closely located to known HPS site in the Eastern Interconnection and created a
+  template to describe the hourly energy generation. The plant level profile is
+  generated considering the local time and daylight-saving schedule of the
+  corresponding bus location.
++ **Conventional hydro**: profiles provided by `ISONE`_, `NYISO`_, `PJM`_ and `SPP`_
+  are used for plants located within the area they cover. For states that are not fully
+  covered by the 4 ISOs, the method that we developed for California, i.e., the monthly
+  net demand in the state is normalized such that its sum is equal to the total monthly
+  hydro generation divided by the available total hydro capacity in the grid model, is
```

Comment:
"...its sum is equal to the total monthly hydro generation divided by the available total hydro capacity in the grid model..." -> "...its sum is equal to the total monthly hydro generation based on EIA form 923 divided by the total hydro capacity in the grid model..."

Also, should note the differences between "states partially covered by the 4 ISOs" vs "states completely outside the 4 ISOs", given in the former case, we also split the total monthly generation proportional to the capacity of hydro plants and only use the portion that is outside the 4 ISOs during the normalization.

Reply:
Done